client/my-sites/stats/post-detail-highlights-section/style.scss

```diff
@@ -111,6 +111,7 @@
 		padding: 0;
 		margin: 0 0 -7px 0;
 		box-shadow: none;
+		overflow: hidden;
```

Comment:
If the overflow is set to `hidden`, the avatar list will be cut vertically. The reported issue seemed to be the unexpected horizontal scroll bar. Is only setting `overflow-y: hidden;` resolving the issue?